.github/workflows/publish-docs-rs.yml

```diff
@@ -36,6 +36,7 @@ jobs:
           path: ./target/doc
       - run: ls -a
       - run: rm -rf ./docs/docs.rs
+      - run: mkdir docs || true # || true to make sure exit code is 0 even if dir exists
```

Comment:
`mkdir -p` also does this

Reply:
yep! Since it wasn't a nested dir, I thought reading `-p` 6 months later would feel more confusing. but I guess the inline comment could have explained that as well :)